back/app/policies/follower_policy.rb

```diff
@@ -33,12 +33,17 @@ def create?
     return false if !user
 
     if user.active? && record.user_id == user.id
-      case record.followable_type
+      policy_class = case record.followable_type
```

Comment:
I would prefer to avoid using constantize to reduce the risk for code injections. A code injection shouldn't be possible in this case, but since the followable type is extracted from the params it feels dangerous.